internal/provider/tag_resource.go

```diff
@@ -53,11 +57,18 @@ func (r *TagResource) Schema(ctx context.Context, req resource.SchemaRequest, re
 			},
 			"tag": schema.StringAttribute{
 				MarkdownDescription: "Tag to apply to the image.",
-				Required:            true,
+				Optional:            true,
 				Validators:          []validator.String{validators.TagValidator{}},
 				PlanModifiers:       []planmodifier.String{stringplanmodifier.RequiresReplace()},
+				DeprecationMessage:  "The `tag` attribute is deprecated. Use `tags` instead.",
+			},
+			"tags": schema.ListAttribute{
+				MarkdownDescription: "Tags to apply to the image.",
+				// TODO: make this required after tag deprecation period.
+				Optional:      true,
+				ElementType:   basetypes.StringType{},
+				PlanModifiers: []planmodifier.List{listplanmodifier.RequiresReplace()},
```

Comment:
Should this have validation that checks uniqueness or that the tag is a valid tag? 🤔